src/panasonicmn_int.cpp

```diff
@@ -720,6 +720,71 @@ namespace Exiv2 {
         return os;
     }  // PanasonicMakerNote::printPitchAngle
 
+/**
+  Chromatic Aberrations info is makernote tag 0x011B. There are 64 bytes of data.
+  The meaning is yet unknown.
+*/
+    std::ostream& PanasonicMakerNote::printChromaticAberration(std::ostream& os, const Value& value, const ExifData*)
```

Comment:
If unknow, I'd remove this special print function for the time being; the bytes are printed anyway.

Reply:
Yes, of course. Would you delete it, oder should I do it? I don't know how to do it since I already made this PR.

Reply:
You can just keep committing changes to your branch, they'll show up here automatically.

Reply:
Hi @greenoid . You can either apply a new commit removing that function, or amend the existing commit (and afterwards do `git push --force`).

Reply:
I applied a new commit removing the function.